frontend/src/services/ApiService.ts

```diff
@@ -19,22 +19,27 @@ export class HttpError extends Error {
  * @returns HttpError if AxiosError, else original error
  */
 export const transformAxiosError = (e: Error): ApiError => {
-  if (axios.isAxiosError(e) && e.response) {
-    const statusCode = e.response.status
-    if (statusCode === 429) {
-      return new HttpError('Please try again later.', statusCode)
-    }
-    if (typeof e.response.data === 'string') {
-      return new HttpError(e.response.data, statusCode)
-    }
-    if (e.response.data?.message) {
-      return new HttpError(e.response.data.message, statusCode)
-    }
-    if (e.response.statusText) {
-      return new HttpError(e.response.statusText, statusCode)
-    }
-
-    return new HttpError(`Http ${statusCode} error`, statusCode)
+  if (axios.isAxiosError(e)) {
+    if (e.response) {
+      const statusCode = e.response.status
+      if (statusCode === 429) {
```

Comment:
minor nit - should we use `http-status-codes` instead of hardcoding 429?

Reply:
actually we could be doing this everywhere in the react app right haha i did a cursory search and we never do this anywhere in `frontend/` :P guess i'll be the first one!

Reply:
updated in [6f93a1b](https://github.com/opengovsg/FormSG/pull/4615/commits/6f93a1bb03d02a8452639b65ab31bd1735ec629e)